src/Pages/RoomDetail/BottomInfo/BottomInfo.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import ReservationTab from './ResevationTab/ReservationTab';
+import ReviewTab from './ReviewTab/ReviewTab';
+import Theme from '../../../Styles/Theme';
+
+const BottomInfo = ({ data }) => {
+  const [currentId, setCurrentId] = useState(1);
+
+  const tabHandler = id => {
+    setCurrentId(id);
+  };
+
+  const TAB_OBJ = {
+    1: <ReservationTab />,
+    2: <ReviewTab />,
+  };
+  return (
+    <BottomInfoWrapper>
+      {/* calender */}
+      <ul className="tabs">
+        {CATEGORY_ARR.map((category, idx) => {
+          return (
+            <Tab
+              key={idx}
+              onClick={() => tabHandler(idx + 1)}
+              isClicked={currentId === idx + 1}
+            >
+              {category}
+            </Tab>
+          );
+        })}
+      </ul>
+      <div className="contents">{TAB_OBJ[currentId]}</div>
+    </BottomInfoWrapper>
+  );
+};
+
+export default BottomInfo;
+
+const CATEGORY_ARR = ['객실안내/예약', '리뷰'];
+
+const BottomInfoWrapper = styled.div`
+  max-width: 1000px;
+  padding: 20px;
+  margin: 0 auto;
+
+  .tabs {
+    display: flex;
+    justify-content: flex-start;
+    color: ${Theme.middleGray};
```

Comment:
```suggestion
    color: ${props => props.theme.middleGray};
```
theme 객체는 themeProvider에서 꺼내어 사용하시면 됩니다. 공식 문서 참조해주세요